packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -39,16 +39,12 @@
                 :data-test-id="testIdForSection('subStatusText', index)">
                 {{ subStatusLine }}
             </div>
-            <div
+            <p
```

Comment:
Should this have stayed as a `div` and the line breaks be implemented in CSS?  It feels odd to use a `p` outside text.

Reply:
This looks ok to me.

Reply:
agreed, I think because I was experimenting with what I wanted to do here I forgot about the surrounding tag. tbh I think it should be 2 `p` tags inside a div. Thanks for the question

Reply:
To me, a `p` is something that defines text content though.  It's akin to punctuation.

Reply:
well the content is `your discount expires on Thursday the 3d of January` as example seems like should be within a `p` tag to me

Reply:
here one is longer format for accessibility and the other is short format to fit the design being given. short `Discount expires 31/01/2023` long accessible one is ` Your discount expires on Tuesday the 31st of January 2023`

Reply:
I think a `p` makes semantic sense here, as long as it's only `span` tags being used to wrap text inside it (which it looks like it is atm)